contracts/build.rs

```diff
@@ -25,7 +25,15 @@ fn main() {
             100 => Address::from_str("1C232F01118CB8B424793ae03F870aa7D0ac7f77").unwrap(),
         },
     );
-    generate_contract("UniswapV2Factory", hashmap! {});
+    generate_contract(
+        "UniswapV2Factory",
+        hashmap! {
+            1 => Address::from_str("5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f").unwrap(),
+            4 => Address::from_str("5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f").unwrap(),
+            100 => Address::from_str("A818b4F111Ccac7AA31D0BCc0806d64F2E0737D7").unwrap(),
```

Comment:
Given that we are prepared for xDAI here, is the `UniswapV2Factory` analogous to the honeyswap contracts on xDAI?

Reply:
I am not sure. I took this part from Felix's PR where he also had to add this contract. It should be this exact contract https://github.com/Uniswap/uniswap-v2-core/blob/master/contracts/UniswapV2Factory.sol just on xdai. Whether honeyswap also uses this or they have their own version I don't know.